openwisp_controller/geo/estimated_location/tasks.py

```diff
@@ -42,6 +42,9 @@ def _create_estimated_location(device_location, location_defaults):
                 f"Estimated location saved successfully for {device_pk}"
                 f" for IP: {ip_address}"
             )
+            send_whois_task_notification(
+                device_pk=device_pk, notify_type="location_created", actor=location
```

Comment:
Why we need location as actor? Also, can you please add ss of sample notification in your PR itself?